### ISSUE TITLE
Names not announced for form fields if their value is not empty

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/test_nvdaTracImport#129]
**Reported by jteh on 20 Mar 2008 02:11 UTC**
When tabbing or navigating to form fields in a virtual buffer, NVDA reads the name of the field (usually obtained from an associated label) prior to the role. However, if a field's value is not empty, the name is not read.

== Reproduceable: always ==

== Steps to Reproduce ==
 1. Go to http://trac.nvda-project.org/.
 2. Press f once to navigate to the search field. Observe that "Search:" is spoken before the "edit" role.
 3. Press enter and type some text.
 4. Press NVDA+space to disable virtual buffer pass-through.
 5. Press control+home to move to the top of the document.
 6. Press f once to navigate to the search field again.

== Actual Results ==
The "Search:" name is not announced in step 6.

== Expected Results ==
The "Search:" name should be announced.

**Comment by michaelDCurran** [comment]
**Comment by mdcurran on 28 Apr 2008 00:43 UTC**
Fixed in Gecko ia2 virtualBuffer backend, included in Misc dependencies 2008-04-28.
